libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -279,6 +279,18 @@ static uint8_t * encodeString( uint8_t * pDestination,
                                const char * pSource,
                                uint16_t sourceLength );
 
+/**
+ * Retrieves and decodes the Remaining Length from the network interface by
+ * reading a single byte at a time.
+ *
+ * @param[in] recvFunc Network interface receive function.
+ * @param[in] networkContext Network interface context to the receive function.
```

Comment:
Nit: really minor but I suppose this param tag should be updated too
```suggestion
 * @param[in] pNetworkContext Network interface context to the receive function.
```

Reply:
Thanks for using the sugguest change featire